aten/src/ATen/native/native_functions.yaml

```
@@ -8142,6 +8142,16 @@
     CPU: searchsorted_cpu
     CUDA: searchsorted_cuda
 
+- func: _convert_coo_to_csr.Tensor(Tensor self, int size, *, bool out_int32=False) -> Tensor
```

Overload name is not needed here.
```suggestion
- func: _convert_coo_to_csr(Tensor self, int size, *, bool out_int32=False) -> Tensor
```